### DISCUSSION TITLE
Dodanie filtra do GET /messaging/threads

### DISCUSSION THREAD

**Discussion by paafcioc22** [allegro/allegro-api#10029] [Suggestions - rejected]
Nie rozumiem dlaczego tak duży zasób jakim są wiadomości nie można pobrać zawężając do klienta czy np daty.
Fajnie gdyby właśnie można było pobrać wszystkie wiadomości dla konkretnego klienta. Jakie są szanse na wprowadzenie takiego rozwiązania?

**Comment by Lukasz-Zurek** [comment]
Zaktualizowaliśmy status sugestii jako "odrzucona", ponieważ nie planujemy sugerowanej zmiany.

W nowej wersji beta.v1 wskazanego zasobu, umożliwimy filtrowanie po page.id, natomiast bez możliwości pobierania wątków dla konkretnego klienta (w tej wersji wdrożyliśmy nowy obiekt "participants").
